doc/release-notes.md

```diff
@@ -1 +1,266 @@
-See https://github.com/bitcoin-core/bitcoin-devwiki/wiki/29.0-Release-Notes-draft
+Bitcoin Core version 29.0 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-29.0/>
+
+This release includes new features, various bug fixes and performance
+improvements, as well as updated translations.
+
+Please report bugs using the issue tracker at GitHub:
+
+  <https://github.com/bitcoin/bitcoin/issues>
+
+To receive security and update notifications, please subscribe to:
+
+  <https://bitcoincore.org/en/list/announcements/join/>
```

Comment:
We should probably drop or update this, given the link to the mailing list on https://bitcoincore.org/en/list/announcements/join/ doesn't work (https://github.com/bitcoin-core/bitcoincore.org/issues/1095).

Reply:
Oh right, that mailing list is dead too. And there's no clear path forward here, probably better to remove it for now.

Reply:
Hopefully we will shortly fix this on the website side.

Reply:
This wasn't removed in #32252, do we want to do it before tag?

Reply:
> This wasn't removed in https://github.com/bitcoin/bitcoin/pull/32252, do we want to do it before tag?

Given how many links like this already exist, and how long release notes stick around, nah, let's leave it. It's better to fix it on the site.